src/my_react/render.ts

```diff
@@ -0,0 +1,85 @@
+const addAttribute = ($elem: HTMLElement, props: JSX.Props, prop: string) => {
+  /* props의 종류에 따라 필요한 처리, 예외 케이스 더 확인되면 추가 필요 */
+  switch (prop) {
+    case 'styles': {
+      Object.keys(props[prop]).forEach((key) => {
+        ($elem.style as any)[key] = props[prop][key];
+      });
+      return;
+    }
+    case 'key':
+      return;
+    case 'children':
+      return;
+    default: {
+      prop.slice(0, 2) === 'on'
+        ? $elem.addEventListener(prop.slice(2).toLowerCase(), props[prop])
+        : $elem.setAttribute(prop, props[prop]);
+      return;
+    }
+  }
+};
+
+const addChildren = ($parent: HTMLElement, children: JSX.Element | JSX.Element[]) => {
+  if (Array.isArray(children)) {
+    // 배열이면 순회하면서 자식 요소 추가
+    children.forEach((child: JSX.Element) => renderDOM($parent, child.type, child.props));
```

Comment:
```suggestion
    children.forEach((child) => renderDOM($parent, child.type, child.props));
```
매개 변수에 타입을 정해놔서 타입 추론이 잘 될 것 같은데 타입 빼면 에러가 생길까요??